desktop/extensions-sdk/architecture/metadata.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
This page doesn't display the table of contents: 
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/15997951/206709478-5003c3e6-b31d-4a6b-9777-29afefdac6d6.png">

Reply:
Oh you are right, me too! @usha-mandya any idea why this is happening?

Reply:
Solved it... there was a missing / at the end of the line in the toc...

Reply:
Looks good now - https://deploy-preview-16289--docsdocker.netlify.app/desktop/extensions-sdk/architecture/metadata/